.github/CODEOWNERS

```
@@ -56,6 +56,14 @@ Tools/c-analyzer/             @ericsnowcurrently
 # dbm
 **/*dbm*                      @corona10 @erlend-aasland @serhiy-storchaka
 
+# Doc/ tools
+Doc/conf.py                   @AA-Turner
+Doc/Makefile                  @AA-Turner
+Doc/make.bat                  @AA-Turner
+Doc/requirements.txt          @AA-Turner
+Doc/_static/**                @AA-Turner
+Doc/tools/**                  @AA-Turner
```

I'll join you for these :)

```suggestion
Doc/conf.py                   @AA-Turner @hugovk
Doc/Makefile                  @AA-Turner @hugovk
Doc/make.bat                  @AA-Turner @hugovk
Doc/requirements.txt          @AA-Turner @hugovk
Doc/_static/**                @AA-Turner @hugovk
Doc/tools/**                  @AA-Turner @hugovk
```